src/components/Dashboard/TodoList.vue

```diff
@@ -615,60 +610,60 @@ export default class TodoList extends Mixins(
   @Action setHasBlockerTask!: ActionBindingIF
 
   /** The Auth Web URL string. */
-  private get authWebUrl (): string {
+  get authWebUrl (): string {
```

Comment:
There is no harm in calling these methods externally so they don't need to be private or protected.